Flutter/calendar/timeslot-views.md

```diff
@@ -530,6 +530,31 @@ Widget build(BuildContext context) {
 
 ![Allday panel Background color](images/getting-started/alldayPanelColor.jpg)
 
+## Number of days in view
+You can customize the days count, by setting the [numberOfDaysInView]() property of `timeSlotViewSettings` in the calendar.
```

Comment:
You can customize the days count by setting the [numberOfDaysInView]() property of `timeSlotViewSettings` in the calendar.

Reply:
I have checked and updated in my latest commit